### DISCUSSION TITLE
加载模型时必须要使用trust_remote_code=True吗？

### DISCUSSION THREAD

**Discussion by spacecat2002** [zai-org/ChatGLM3#820] [Q&A] [answered]
我已经在model文件夹中下载好了模型，但载入本地模型时仍然提示让我使用trust_remote_code=True
![PixPin_2024-02-05_15-12-04](https://github.com/THUDM/ChatGLM3/assets/77487702/ddb5978e-4452-4ab2-bf25-fe750548dab4)
![PixPin_2024-02-05_15-12-27](https://github.com/THUDM/ChatGLM3/assets/77487702/43e88ea5-07a0-498c-a637-d50714e6ca83)

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
必须
